### PR TITLE
feat: adds dual write docs to android sdk

### PR DESCRIPTION
**Pull request by kevinperaza**
## Description

- feat: adds dual write docs to android sdk

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [X] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: docs/sdks/mobile/android/services.mdx [hunk 1]
```diff
@@ -135,3 +136,137 @@ val proxyResponse = bt.proxy.post(proxyRequest)
...
+private val bt = BasisTheoryElements.builder()
+    .apiUrl(BuildConfig.BASIS_THEORY_API_URL)
+    .apiKey(BuildConfig.BASIS_THEORY_API_KEY)
+    .build()
+...
+
+bt.client.get("https://api.psp.com/v1/payment_methods/id",
+    mapOf(
+        "Authorization" to "Bearer ...",
+        "Content-Type" to "application/x-www-form-urlencoded"
```
Comment (dhudec) [inline docs/sdks/mobile/android/services.mdx:248]: We should probably remove Content-Type on this GET request

File: docs/sdks/mobile/android/types.mdx [hunk 2]
```diff
@@ -261,9 +277,10 @@ that are supported on the base [TextElement](#textelement).
 ### Additional Methods Supported
 
 | Signature                                                                      | Description                                                                           |
-|--------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| ------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------- |
 | `month(): ElementValueReference`                                               | Returns a value reference you can use when tokenizing without touching the raw value. |
 | `year(): ElementValueReference`                                                | Returns a value reference you can use when tokenizing without touching the raw value. |
+| `format(dateFormat: String): ElementValueReference`                            | Returns a value reference you can use when tokenizing without touching the raw value. |
```
Comment (dhudec) [inline docs/sdks/mobile/android/types.mdx:283]: I'm just noticing the description for month(), year(), and format() are all the same. We should probably add a few words to emphasize the difference. Also noticing the mention of "you can use when tokenizing" which I think was true at first but we can use these references for other purposes now too, so I'd consider removing that too. Maybe something like:

```
| `month(): ElementValueReference`                                               | Returns a value reference to the two-digit expiration month. |
| `year(): ElementValueReference`                                                | Returns a value reference to the four-digit expiration year. |
| `format(dateFormat: String): ElementValueReference`                            | Returns a value reference to the expiration date formatted according to the provided `dateFormat`. |
```

File: docs/sdks/mobile/android/services.mdx (hunk 1 above)
Comment (kevinperaza) [reply docs/sdks/mobile/android/services.mdx]: Removed!

File: docs/sdks/mobile/android/types.mdx (hunk 2 above)
Comment (kevinperaza) [reply docs/sdks/mobile/android/types.mdx]: Updated, it's formatted to two digits

**Review by dhudec — APPROVED** [review]

**Pull request merged**